packages/mobile/src/screens/profile-screen/ProfileHeader/SocialLink.tsx

```diff
@@ -1,7 +1,7 @@
 import { useCallback, useState } from 'react'
 
 import type { StyleProp, ViewStyle } from 'react-native'
-import { View } from 'react-native'
+import type { Nullable } from '~/utils'
```

Comment:
sorry mind updating this to @audius/common/utils

Reply:
doing.